index.js

```diff
@@ -5,4 +5,11 @@ var
 // Adds on the fly methods promisification
 Kuzzle.prototype.bluebird = bluebird;
 
+if (typeof window !== 'undefined' && typeof BUILT === 'undefined') {
+  throw new Error('It looks like you are using the Nodejs version of Kuzzle SDK ' +
+               'in a browser. ' +
+               'It is strongly recommended to use the browser-specific build instead. ' +
```

Comment:
Since we now throw an error, we are more than "strongly recommending" to use the built-version of the SDK. :-)